hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/converter/MessageConverterFactory.java

```diff
@@ -0,0 +1,20 @@
+package pl.allegro.tech.hermes.consumers.consumer.converter;
+
+import org.apache.avro.Schema;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.common.message.converter.AvroToJsonConverter;
+
+import static pl.allegro.tech.hermes.api.Topic.ContentType.AVRO;
+
+public class MessageConverterFactory {
+
+    private final DefaultMessageConverter defaultMessageConverter = new DefaultMessageConverter();
```

Comment:
Here you can use the same patter I wrote in comment in MessageContentWrapperProvider